### PR TITLE
added MSLP example

### PR DESCRIPTION
**Pull request by kgoebber**
Adding a simple surface plot using archived GFS analysis files. Plots MSLP, 2m Temp (F), and 10m Wind Barbs (kt)

### CODE REVIEW

File: examples/MSLP_temp_winds.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+
+###############################################
+# Create NCSS object to access the NetcdfSubset
+# ---------------------------------------------
+# Data from NCEI GFS 0.5 deg Analysis Archive
+
+base_url = 'https://www.ncei.noaa.gov/thredds/ncss/grid/gfs-g4-anl-files/'
+dt = datetime(2018, 1, 4, 12)
+ncss = NCSS('{}{dt:%Y%m}/{dt:%Y%m%d}/gfsanl_4_{dt:%Y%m%d}_{dt:%H}00_000.grb2'.format(base_url, dt=dt))
```
Comment (jrleeman) [inline examples/MSLP_temp_winds.py]: Will need to wrap this.

File: examples/MSLP_temp_winds.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+#
+# MSLP: Pressure_reduced_to_MSL_msl (time, lat, lon)
+# 2m Temp: Apparent_temperature_height_above_ground (time, level, lat, lon)
+# 10m Wind: u/v-component_of_wind_height_above_ground (time, level, lat, lon)
+#
+# Height above ground Temp from GFS has one level (2m)
+# Height above ground Wind from GFS has three levels (10m, 80m, 100m)
+
+# Pull out variables you want to use
+mslp_var = data.variables['Pressure_reduced_to_MSL_msl']
```
Comment (jrleeman) [inline examples/MSLP_temp_winds.py]: Could we combine this and the block below to `mslp_var = data.variables['Pressure_reduced_to_MSL_msl'][:].squeeze()`? It's a heck of a line, but reduces the total SLOC here.

File: examples/MSLP_temp_winds.py (hunk 1 above)
Comment (kgoebber) [reply examples/MSLP_temp_winds.py]: done

File: examples/MSLP_temp_winds.py (hunk 2 above)
Comment (kgoebber) [reply examples/MSLP_temp_winds.py]: done

**Pull request merged**